src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationTestFixture.cs

```diff
@@ -84,9 +85,12 @@ protected WebApplicationTestFixture(string solutionRelativePath)
         /// found traversing up the folder hierarchy from the test execution folder is considered as the base path.</param>
         protected WebApplicationTestFixture(string solutionSearchPattern, string solutionRelativePath)
```

Comment:
Should we make these public now that you aren't expected to subclass this often?